.github/workflows/tests.yml

```diff
@@ -30,8 +30,8 @@ jobs:
 
         exclude:
           # Laravel 8.0 has type incompatibilities with PHP 8.1.
+          # And Full-text indexes and scopes are supported since Laravel 8.79
           - laravel: 8.*
-            php: 8.1
             dependency-version: prefer-lowest
```

Comment:
Tests failed because Full-text indexes and scopes are only [available since Laravel 8.79](https://github.com/laravel/framework/releases/tag/v8.79.0) and the `prefer-lowest` flag was allowing composer to install older versions.

I think limiting support and tests to more recent versions of Laravel is not only fine, but preferred. Updating between Laravel minor releases is completely painless, so users of the package shouldn't have issues with that.

We could also increase the package minimum required version of Laravel to 8.79.0, though I don't see it as strictly necessary since anyone using Full-text methods would already be using Laravel +8.79.

@carlsverre let me know what you think

Reply:
In that case, please skip tests which require higher laravel versions. Something like:
```php
        if (version_compare(Application::VERSION, '8.12.0', '<=')) {
            // fulltext not added until later on in laravel 8 releases
            $this->markTestSkipped('requires higher laravel version');

            return;
        }
```

Reply:
Yeah, that would be a better approach. Let me try it

Reply:
Works perfectly, now we're ready ready